src/doc/src/reference/unstable.md

```diff
@@ -1695,21 +1695,38 @@ This feature moves the handwritten completion scripts to Rust native, making it
 easier for us to add, extend and test new completions. This feature is enabled with the
 nightly channel, without requiring additional `-Z` options.
 
+Areas of particular interest for feedback
```

Comment:
We don't really document the feedback process for other unstable features. Why do we decide to do it here?

Reply:
- Most of this feature lives outside of the cargo's repo so usual processes will likely add more friction
- There are enough corner cases with this that I want to be very cautious in saying this is good enough to replace the existing completions so I need to depend more on broad feedback than we typically do

Reply:
Fair enough. We might also want a copy of this process in the tracking issue.